tests/test_spacy_udpipe.py

```diff
@@ -33,14 +33,17 @@ def test_pipe(lang: str) -> None:
     nlp = load(lang=lang)
     assert nlp._meta["lang"] == f"udpipe_{lang}"
 
-    text = "spacy-udpipe package now support multiprocess execution."
+    text = "spacy-udpipe still does not support multiprocess execution."
     doc = nlp(text)
+    del nlp
```

Comment:
This line makes the test a proper one!

Reply:
I don't quite get it. You don't really test whether multiprocesing is enabled here or not. And the only way to do so, would be to check whether you set the n_process in the subclass. You could do that with some mocks, such that mocks are returned when calling with n_processes != 1.Up to you.

Reply:
This avoids the case in which you process the same texts with `nlp.call` and then with `nlp.pipe` using the same `nlp` instance (when multiprocessing 'falsely' works). In that case, multiprocessing raises no errors as the `nlp.vocab` already contains all previously unknown token attributes, but multiprocessing also provides no benefits in this case.